sbi/utils/sbiutils.py

```diff
@@ -21,10 +17,20 @@
 from torch import nn as nn
 from torch import ones, zeros
 from tqdm.auto import tqdm
+import warnings
 
 
 def x_shape_from_simulation(batch_x: Tensor) -> torch.Size:
-    assert batch_x.ndim == 2, "Simulated data must be a batch with two dimensions."
+    ndims = batch_x.ndim
+    assert ndims >= 2, "Simulated data must be a batch with at least two dimensions."
+
+    if ndims >= 2:
```

Comment:
If it has shape `(1, x_dim)`, wouldnt this trigger a warning? It should be `ndims>3`, no?